### ISSUE TITLE
Remove authentication keys from code

### ISSUE THREAD

**Issue by akshayagarwal** [bilalq/Tranquillity-Editor#11]
I spotted an authentication key for Lymbix API in the /application/controllers/home.php file. Please remove it/ substitute with dummy text, for obvious reasons of misuse. However, just making a new commit with the key substituted will not actually remove the key from public view since we would still be able to see it in the commit history. Its better if you could actually invalidate that key & get new keys from Lymbix .

**Comment by bilalq** [comment]
Yeah, this is should be removed. We're not using Lymbix at all anymore, so this shouldn't be too much of a problem. 

@demodesigns 
Can you invalidate that key? Get a new one if you still want to use it.
